core/cli/templates/homeTemplate.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {homeViewTemplate, homePageTemplate};
+
+function homeViewTemplate() {
+    let template =
+
+`import React, { Component } from 'react';
+
+class HomeView extends Component {
+    constructor(props) {
```

Comment:
I've checked what kind of components create-react-app scaffolds. It creates a ES6 class component without constructor and without defaultProps. I'd maybe do the same. In general I would tend to scaffold only a minimal amount of code. Feel free to contradict

Reply:
I will certainly follow your lead on this.  In my limited experience I would think you would want the constructor function with state by default, but again, I haven't built a lot of React apps, so I will go ahead and drop it and defaultProps for now.  If at some point we decide we want to keep it (like if we end up wanting to load something in state for the entry point of the site) we can just add it back.